### PR TITLE
Fix typos & wording in README + fix of book compilation errors

### PR DESCRIPTION
**Pull request by dev99problems**
### Description
This brings some typos adjustments and wording updates to `README.md` to make wording there a bit more formal
and a bit less **authorish**.

Also fixes **2** books **generation errors**:
* including non-existing `contributors.txt` file — original eng. book now is providing a link to it in `contributors.asc`, instead or **including it**, but the fun part — I couldn't find the `contributors.txt` file itself in eng. repo either, so just switching from `including` to `link` worked pretty well
* fixes the `asciidoc` formatting issue in `book/06-github/3-maintaining.asc` section

### CODE REVIEW

**Comment by dev99problems** [comment]
@burlakvo , @hedrok folks, if you'll find time to go through and provide a feedback — I'd be thankful

File: book/contributors.asc [hunk 1]
```diff
@@ -5,6 +5,6 @@
 
 [source,tabsize=8]
 ----
-include::contributors.txt[]
```
Comment (burlakvo) [inline book/contributors.asc]: It's generated on the go by `Rakefile`, so there is no need for such changes

**Review by burlakvo — COMMENTED** [review]
@dev99problems, thanks for contributing!

There is one small notice regarding `contributors.txt` and in other files, I could suggest rewording "про**е**кт" into "про**є**кт"

_I haven't the ability to approve PRs, only commenting =)_

File: book/contributors.asc (hunk 1 above)
Comment (dev99problems) [reply book/contributors.asc]: Thanks for mentioning this!
I couldn't make `bundle install` land all the deps normally on ruby@3.2.2, for some reason there's a problem with 
kindlegen@3.1.1 gem on my MacOS setup, so I wasn't able to run `bundle exec rake` normally.

Reverting this one

**Comment by dev99problems** [comment]
> @dev99problems, thanks for contributing!
> 
> There is one small notice regarding `contributors.txt` and in other files, I could suggest rewording "про**е**кт" into "про**є**кт"
> 
> _I haven't the ability to approve PRs, only commenting =)_

Was thinking about `проєкт` as well, because seems like since 2019 it is the proper & official spelling.
Will update in `README.md`

File: book/contributors.asc (hunk 1 above)
Comment (dev99problems) [reply book/contributors.asc]: Reverted

**Comment by dev99problems** [comment]
@hedrok якщо буде змога, могли б подивитись ці зміни?

**Comment by hedrok** [comment]
Дякую, залив.
